aladdin-container.sh

```diff
@@ -100,17 +100,15 @@ function environment_init() {
 
         if "$IS_LOCAL"; then
             mkdir -p $HOME/.kube/
-            cp $HOME/.kube_local/config $HOME/.kube/config
-            # Replace e.g. /Users/ptr/.minikube with /root/.minikube in the minikube conf dir path
-            # (since that's where it will be mounted within the aladdin container)
-            sed 's|: \?.*\.minikube|: /root/.minikube| ; /: \/root\/.minikube/ s|\\|/|g' $HOME/.kube_local/config > $HOME/.kube/config
+            sed "s;https://0.0.0.0:$K3D_API_PORT;https://host.docker.internal:$K3D_API_PORT;g" $HOME/.kube_local/config > $HOME/.kube/config
```

Comment:
this is the address of the apiserver from within the aladdin docker container (host.docker.internal)